[MTM-64334] Document tenant option `sso-redirect-default-application`… (#3592)
SoftwareAG/c8y-docs: content/authentication/sso-bundle/sso-troubleshooting.md

  - platform_administration
---

> ### Inspect token content

Reviewer: ```suggestion
### Inspect token content {#inspect-token-content}
```

It can be particularly helpful to inspect the content of the authorization token sent to the platform as some of its
fields contain the information required for the correct configuration described above.

> In the Administration application, click **Accounts** > **Audit logs** and filter by the category "Single sign-on" and

Reviewer: ```suggestion
In the Administration application, click **Accounts** > **Audit logs**, filter by the type "Single sign-on" and
```

> look for entries "Json web token claims".

Reviewer: ```suggestion
look for entries with "JSON web token claims".
```

The contexts of the token will be presented in JSON format.

![Audit token content](/images/users-guide/Administration/admin-sso-audit-token.png)

> ### Enforce use of tenant domain in SSO login

Reviewer: ```suggestion
### Enforce the use of the tenant domain in SSO login {#enforce-use-of-tenant-domain}
```

> When a tenant is configured to use the `tenantId` in the `baseUrl` (instead of a tenant domain), users may experience

Reviewer: ```suggestion
If a tenant is configured to use the `tenantId` in the `baseUrl` (instead of a tenant domain), users may experience
```

unexpected redirect behavior after authenticating via SSO.

> If a user opens an application by entering its URL directly and initiates login via SSO, they may not be returned to the

Reviewer: ```suggestion
If users opens an application by entering its URL directly and initiates login via SSO, they may not be returned to the
```

> If users opens an application by entering its URL directly and initiates login via SSO, they may not be returned to the

Reviewer 2: should be 'If a user opens ..'

> If users opens an application by entering its URL directly and initiates login via SSO, they may not be returned to the

Reviewer: ```suggestion
If a user opens an application by entering its URL directly and initiates login via SSO, they may not be returned to the
```

intended application after authentication. Instead, they may be redirected to the default application configured for the
tenant.

> This can be fixed by setting the following tenant options:

Reviewer: ```suggestion
This can be changed by setting the following tenant options:
```

- category: `sso`
- key: `sso-redirect-default-application`
- value: `false`

Setting this tenant option to `false` will enforce the use of the tenant domain during SSO login. This results in:

- Correctly scoped cookies being set for the intended application, ensuring the user is redirected back to the original application after successful authentication.
- Support for SSL certificates using **Subject Alternative Name (SAN)**, eliminating the need for wildcard or tenant-specific certificates.

> ##### When to use:

Reviewer: ```suggestion
**When to use:**
```
